sdk/identity/azure-identity/pom.xml

```diff
@@ -27,7 +27,7 @@
     <dependency>
       <groupId>com.azure</groupId>
       <artifactId>azure-core</artifactId>
-      <version>1.7.0</version> <!-- {x-version-update;com.azure:azure-core;dependency} -->
+      <version>1.8.0</version> <!-- {x-version-update;com.azure:azure-core;dependency} -->
```

Comment:
This change should have already been in place from [this PR](https://github.com/Azure/azure-sdk-for-java/pull/14803/files#diff-b1a225c9ee81df78dc378973728e3ca6L30)